docs/configuring.md

```diff
@@ -573,6 +573,29 @@ questions with default answers.
     copier -fd 'user_name=Manuel Calavera' copy template destination
     ```
 
+As alternative you can also pass the path to yaml file that contains your data. This
+cannot be used in the API.
```

Comment:
A few minor linguistic corrections.

And how about a little rewording in the second sentence?

```suggestion
As an alternative you can also pass the path to a YAML file that contains your data.
Passing a data file is only supported through the CLI.
```